dotcom-rendering/src/types/front.ts

```diff
@@ -289,9 +309,7 @@ export type DCRFrontCard = {
 	byline?: string;
 	showByline?: boolean;
 	avatarUrl?: string;
-	mediaType?: MediaType;
-	mediaDuration?: number;
-	showMainVideo: boolean;
```

Comment:
I could not find examples of when we want to hide the main video, so have removed as a simplication step.